### PR TITLE
add SetWMClass RemoveAccelGroup ActivateFocus ActivateDefault

### PR DESCRIPTION
**Pull request by jonib3k**

### CODE REVIEW

File: gtk/gtk.go [hunk 1]
```diff
@@ -1287,7 +1287,13 @@ func (v *Window) SetTitle(title string) {
 	C.gtk_window_set_title(WINDOW(v), gstring(ptr))
 }
 
-// gtk_window_set_wmclass
+func (v *Window) SetWMClass(wmclass_name string, wmclass_class string) {
```
Comment (mattn) [inline gtk/gtk.go]: please rename parameters to name and clazz.

File: gtk/gtk.go [hunk 2]
```diff
@@ -1301,9 +1307,17 @@ func (v *Window) AddAccelGroup(group *AccelGroup) {
 	C.gtk_window_add_accel_group(WINDOW(v), group.GAccelGroup)
 }
 
-// gtk_window_remove_accel_group
-// gtk_window_activate_focus
-// gtk_window_activate_default
+func (v *Window) RemoveAccelGroup(accel_group *AccelGroup) {
```
Comment (mattn) [inline gtk/gtk.go]: please rename this parameter to `agroup`.

**Comment by jonib3k** [comment]
also renamed parameter in AddAccelGroup to be consistent with RemoveAccelGroup

**Comment by mattn** [comment]
Thanks.

**Pull request merged**